boards/native_posix_64.conf

```diff
@@ -20,6 +20,10 @@ CONFIG_LOG_BACKEND_NATIVE_POSIX=y
 CONFIG_SHELL_LOG_BACKEND=n
 CONFIG_LOG_MODE_IMMEDIATE=y
 
+CONFIG_AOS_LOG_BACKEND_FS_DIR=".aos/logs"
+CONFIG_AOS_LOG_BACKEND_FS_FILE_SIZE=10240
```

Comment:
We can set default file size, let's say 16k and have 8 files rotation and use it for all platforms.